### PR TITLE
[DOCFIX] Update cn version of Spark on Kubernetes doc

### PR DESCRIPTION
**Pull request by 007DXR**
What changes are proposed in this pull request?
Update cn version of Spark on Kubernetes doc.

Why are the changes needed?
There is no corresponding Chinese documentation for upgrade.

Does this PR introduce any user facing changes?
More Chinese users can access Alluxio documentation more easily.

### CODE REVIEW

File: docs/cn/kubernetes/Spark-On-Kubernetes.md [hunk 1]
```diff
@@ -151,41 +134,42 @@ $ ./bin/spark-submit --master k8s://https://<kubernetes-api-server>:8443 \
...
 ```console
 rm spark-2.4.4-bin-hadoop2.7/jars/kubernetes-client-*.jar
 wget https://repo1.maven.org/maven2/io/fabric8/kubernetes-client/4.4.2/kubernetes-client-4.4.2.jar 
 cp kubernetes-client-4.4.2.jar spark-2.4.4-bin-hadoop2.7/jars
 ```
-然后编译`spark-alluxio`镜像，并分发到所有节点。
+然后构建`spark-alluxio`镜像并分发到所有节点。
 
-### 服务帐户没有访问权限
+### Service account does not have access
```
Comment (Jackson-Wang-7) [inline docs/cn/kubernetes/Spark-On-Kubernetes.md:170]: why not translate this sentense into Chinese?

File: docs/cn/kubernetes/Spark-On-Kubernetes.md [hunk 2]
```diff
@@ -195,5 +179,4 @@ Message: Forbidden!Configured service account doesn't have access. Service accou
 pods "spark-alluxiolatest-exec-1" is forbidden: User "system:serviceaccount:default:default" \
 cannot delete resource "pods" in API group "" in the namespace "default".
 ```
-
-你应该参考[创建服务帐户]({{ '/en/kubernetes/Spark-On-Kubernetes.html#create-the-service-account-optional' | relativize_url}}确保有正确访问权限。
+您应该通过[创建服务帐户]({{ '/cn/kubernetes/Spark-On-Kubernetes.html#create-the-service-account-optional' | relativize_url }})来确保账户具有合理的访问权限。
```
Comment (Jackson-Wang-7) [inline docs/cn/kubernetes/Spark-On-Kubernetes.md:182]: recommend to keep the blank line

File: docs/cn/kubernetes/Spark-On-Kubernetes.md [hunk 3]
```diff
@@ -151,41 +134,42 @@ $ ./bin/spark-submit --master k8s://https://<kubernetes-api-server>:8443 \
...
 ```
 WARN ExecutorPodsWatchSnapshotSource: Kubernetes client has been closed
 ...
 ERROR SparkContext: Error initializing SparkContext.
 io.fabric8.kubernetes.client.KubernetesClientException
 ```
-
-这可能是由于一个[已知问题](https://issues.apache.org/jira/browse/SPARK-28921)导致的，可以通过将`kubernetes- client.jar`升级至4.4.x来解决。
-您可以在编译`spark-alluxio`镜像之前通过更新`kubernetes-client-xxjar`来修补docker镜像。
+这可能是由某个[已知问题](https://issues.apache.org/jira/browse/SPARK-28921)导致，该问题可以通过将 `kubernetes-client.jar`升级到4.4.x来解决。您可以在构建`spark-alluxio`镜像之前通过更新`kubernetes-client-x.x.jar`来修补docker镜像。
```
Comment (Jackson-Wang-7) [inline docs/cn/kubernetes/Spark-On-Kubernetes.md:161]:
```suggestion
这可能是由一个[已知问题](https://issues.apache.org/jira/browse/SPARK-28921)导致，该问题可以通过将 `kubernetes-client.jar`升级到4.4.x来解决。您可以在构建`spark-alluxio`镜像之前通过更新`kubernetes-client-x.x.jar`来修补docker镜像。
```

**Review by Jackson-Wang-7 — COMMENTED** [review]
mostly LGTM

**Review by Jackson-Wang-7 — APPROVED** [review]
lgtm

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**